packages/server/lib/plugins/index.coffee

```diff
@@ -68,6 +68,7 @@ module.exports = {
 
       handleError = (err) ->
         debug("plugins process error:", err.stack)
+        return if not pluginsProcess ## prevent repeating this in case of multiple errors
```

Comment:
I double-checked, it's just the same error getting handled twice for some odd reason. Can only reproduce in CI: https://circleci.com/gh/cypress-io/cypress/147087?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Thought it had to do with having 2 'error' listeners, but it still happens with just 1:

https://github.com/cypress-io/cypress/blob/67997628a95fc01f35ee5081afa3e779cc688ff2/packages/server/lib/plugins/index.coffee#L77-L78

Since it's the same error, I don't think it's cause for concern.